README.md

```diff
@@ -29,6 +29,7 @@ npm install vue-compose-promise
 <script>
 import { createComponent } from '@vue/composition-api'
 import { usePromise } from 'vue-compose-promise'
+import { fetchUsers } from './data';
```

Comment:
I rather leave the import out and put a comment saying `fetchUsers` does an api call and returns the Promise